cmd/wego/main.go

```diff
@@ -38,7 +74,17 @@ func configureLogger() {
 }
 
 var ApplicationCmd = &cobra.Command{
-	Use:  "app [subcommand]",
+	Use:   "app [subcommand]",
+	Short: "Manages your applications",
+	Long:  `
+  Add:    Add an application to wego control,
+  Status: Get status of application under wego control`,
+	Example:`
+  # Add an application to wego
+  wego app add <myapp>
```

Comment:
This either needs a dir or a url. Maybe: `wego app add . --name=<myapp>` or just say `wego app add <dir>`